grails-app/domain/org/pih/warehouse/requisition/RequisitionItem.groovy

```diff
@@ -21,9 +21,13 @@ import org.pih.warehouse.product.Category
 import org.pih.warehouse.product.Product
 import org.pih.warehouse.product.ProductGroup
 import org.pih.warehouse.product.ProductPackage
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
 
 class RequisitionItem implements Comparable<RequisitionItem>, Serializable {
 
+    private static final transient Logger log = LoggerFactory.getLogger(RequisitionItem)
```

Comment:
There should be an an implicit logger on all services, controller, and domain classes so you can probably remove this. Not sure why I used println here.

Reply:
There should be, but the implicit logger doesn't come up during unit tests, which is why the previous PR failed tests (cf. f40f82b146). I have no idea why it doesn't under test (an implicit logger _is_ created when the application runs), but being explicit here fixes tests and also works properly when the app runs. That may be why you put a println here (there's only one other log statement in the whole domain/ directory)

PS this is me looking forward to grails 3